app/views/quotes/index.html.erb

```diff
@@ -1,15 +1,20 @@
 <h1>Pithy quotes</h1>
 
 <% @quotes.each do |quote| %>
-  <section class="quote-container">
+<section class="quote-container">
     <blockquote class="quote">
-      <%= quote.text %>
-      <footer class="author">
-        <%= quote.author %>
-      </footer>
+        <%= quote.text %>
+        <footer class="author">
+            <%= quote.author %>
+        </footer>
     </blockquote>
     <div class="created">
-      <%= time_ago_in_words(quote.created_at) %> ago
+        <% if quote.created_at %>
```

Comment:
Oh, I also noticed -- the data is null in a few cases -- can you fix that in the production.sql file? thanks!